xgux.h

```diff
@@ -91,6 +91,20 @@ void xgux_set_clear_rect(unsigned int x, unsigned int y,
     pb_end(p);
 }
 
+#define NV097_SET_CONTROL0_TEXTUREPERSPECTIVE 0x100000
+
+XGUX_API
+void xgux_set_depth_range(float znear, float zfar) {
+    uint32_t *p = pb_begin();
+    uint32_t control0 = NV097_SET_CONTROL0_TEXTUREPERSPECTIVE | NV097_SET_CONTROL0_STENCIL_WRITE_ENABLE;
+    p = push_command_parameter(p, NV097_SET_CONTROL0, control0);
+    p = push_command_parameter(p, NV097_SET_ZMIN_MAX_CONTROL, 1);
+    p = push_command_parameter(p, NV097_SET_COMPRESS_ZBUFFER_EN, 1);
```

Comment:
`push_command_parameter()` isn't used elsewhere in xgux as far as I can tell.
While it's a style break, I on the other hand can't think of a time when these three registers would be controlled individually.

Reply:
`pb_push1()` then?

Reply:
Sorry, I was vague. I should've been clearer that I meant that all other functions seem to call XGU functions rather than `push_X`. I very much prefered `push_command_parameter()` over `pb_push1()`.

Reply:
I'm not quite sure what needs to be done here then. (If anything?)

Reply:
Yeah, nothing really. Was more meant as a question/discussion starter.